package.json

```diff
@@ -11,6 +11,7 @@
     "build:es": "rm -rf es && cross-env BABEL_ENV=es babel src -d es --extensions .ts,.tsx --ignore src/**/*.test.tsx",
     "build:ts": "tsc --rootDir src -d --declarationDir lib && tsc --rootDir src -d --declarationDir es",
     "build:icon": "babel-node --extensions .ts,.tsx src/Utils/Icon/generateIcons.ts",
+    "format": "prettier --write src/**/*.{ts,tsx} stories/**/*.{ts,tsx}",
```

Comment:
in what situation should we run `format`?

Reply:
It was deleted in a later commit, I was using it to fix the prettier's errors.

Reply:
why you keep `format:icon`?

Reply:
Because the Icon Components (under `src/General/Icon/components`) are automatically generated without formatting, and thus them will cause `prettier/prettier`'s error. I'd like to fomat it by script rather than setting ignore them in `prettier/prettier` rule.